helm/coder/values.yaml

```diff
@@ -285,7 +285,7 @@ coder:
     # coder.service.prometheusNodePort -- Enabled if coder.service.type is set
     # to NodePort. If not set, Kubernetes will allocate a port from the default
     # range, 30000-32767. The "prometheus-http" port on the coder service is
-    # only exposed if CODER_PROMETHEUS_ENABLE is set to true.
+    # not exposed if CODER_PROMETHEUS_ENABLE is set to true.
```

Comment:
This value should be dropped entirely, right?

Reply:
Dropped!